### ISSUE TITLE
BatchNorm solution is only correct for training

### ISSUE THREAD

**Issue by rudolfwilliam** [duoan/TorchCode#5]
The provided BatchNorm solution is correct only during training. However, it is typical for interviewers to ask how batch norm is handled during training vs. inference. The current solution does not reflect this circumstance. I believe it would not be too much effort to incorporate this.
